test/src/e2e_vm_tests/test_programs/should_fail/different_contract_caller_types/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
+name = 'core'
+dependencies = []
+
+[[package]]
+name = 'different_contract_caller_types'
```

Comment:
Can you delete and regenerate the lockfiles for the tests here? Right now they're still locked to using the online tagged.